Fluent Interfaces/src/Application.java

```diff
@@ -0,0 +1,64 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Application {
+	
+	private List<Document<Integer>> documents = new ArrayList<>();
+
+	public Application() {
```

Comment:
With fluent interfaces but without non-static initializer block. For that there's a PMD warning.